Makefile

```diff
@@ -53,6 +53,10 @@ markdown-links-lint:
 check-configuration-keys:
 	./ci/check_configuration_keys.py --values deploy/helm/sumologic/values.yaml --readme deploy/helm/sumologic/README.md
 
+.PHONY: check-dependencies
+check-dependencies:
```

Comment:
on Vagrant this target doesn't work:
```
$ make check-dependencies
python ./ci/check_dependencies/main.py
make: python: Command not found
make: *** [Makefile:58: check-dependencies] Error 127
```

Reply:
Ok, I see readme below :)

Reply:
I think you have an outdated machine. Do a `vagrant destroy` and `vagrant up` and it should work.